2022-01-09 17:39:04,375 INFO    MainThread:14777 [wandb_setup.py:_flush():71] setting env: {}
2022-01-09 17:39:04,375 INFO    MainThread:14777 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-09 17:39:04,375 INFO    MainThread:14777 [wandb_init.py:_log_setup():371] Logging user logs to /content/tbcnn/wandb/run-20220109_173904-2o1osl1c/logs/debug.log
2022-01-09 17:39:04,375 INFO    MainThread:14777 [wandb_init.py:_log_setup():372] Logging internal logs to /content/tbcnn/wandb/run-20220109_173904-2o1osl1c/logs/debug-internal.log
2022-01-09 17:39:04,375 INFO    MainThread:14777 [wandb_init.py:init():404] calling init triggers
2022-01-09 17:39:04,376 INFO    MainThread:14777 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-01-09 17:39:04,376 INFO    MainThread:14777 [wandb_init.py:init():460] starting backend
2022-01-09 17:39:04,376 INFO    MainThread:14777 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-09 17:39:04,393 INFO    MainThread:14777 [backend.py:ensure_launched():216] starting backend process...
2022-01-09 17:39:04,404 INFO    MainThread:14777 [backend.py:ensure_launched():222] started backend process with pid: 14790
2022-01-09 17:39:04,410 INFO    MainThread:14777 [wandb_init.py:init():469] backend started and connected
2022-01-09 17:39:04,422 INFO    MainThread:14777 [wandb_init.py:init():533] updated telemetry
2022-01-09 17:39:04,489 INFO    MainThread:14777 [wandb_init.py:init():563] communicating current version
2022-01-09 17:39:05,541 INFO    MainThread:14777 [wandb_init.py:init():568] got version response 
2022-01-09 17:39:05,541 INFO    MainThread:14777 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-09 17:39:05,766 INFO    MainThread:14777 [wandb_init.py:init():606] starting run threads in backend
2022-01-09 17:39:08,515 INFO    MainThread:14777 [wandb_run.py:_console_start():1810] atexit reg
2022-01-09 17:39:08,515 INFO    MainThread:14777 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-09 17:39:08,516 INFO    MainThread:14777 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-09 17:39:08,521 INFO    MainThread:14777 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-09 17:39:08,521 INFO    MainThread:14777 [wandb_init.py:init():633] run started, returning control to user process
2022-01-09 17:39:17,077 INFO    MainThread:14777 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-09 17:39:17,078 INFO    MainThread:14777 [wandb_run.py:_restore():1752] restore
2022-01-09 17:39:19,559 INFO    MainThread:14777 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 686
  total_bytes: 686
}

2022-01-09 17:39:19,821 INFO    MainThread:14777 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 686
  total_bytes: 686
}

2022-01-09 17:39:20,799 INFO    MainThread:14777 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 686
  total_bytes: 11144
}

2022-01-09 17:39:20,901 INFO    MainThread:14777 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 686
  total_bytes: 11144
}

2022-01-09 17:39:21,004 INFO    MainThread:14777 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 686
  total_bytes: 11144
}

2022-01-09 17:39:21,106 INFO    MainThread:14777 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11144
  total_bytes: 11144
}

2022-01-09 17:39:21,208 INFO    MainThread:14777 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11144
  total_bytes: 11144
}

2022-01-09 17:39:21,311 INFO    MainThread:14777 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11144
  total_bytes: 11144
}

2022-01-09 17:39:21,413 INFO    MainThread:14777 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11144
  total_bytes: 11144
}

2022-01-09 17:39:21,515 INFO    MainThread:14777 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11144
  total_bytes: 11144
}

2022-01-09 17:39:22,228 INFO    MainThread:14777 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11144
  total_bytes: 11144
}

2022-01-09 17:39:22,702 INFO    MainThread:14777 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11144
  total_bytes: 11144
}
local_info {
}

2022-01-09 17:39:24,301 INFO    MainThread:14777 [wandb_run.py:_append_files():2180] logging synced files
